services-core/src/gas_price.rs

```diff
@@ -27,10 +28,24 @@ pub trait GasPriceEstimating: Send + Sync {
 pub async fn create_estimator(
     http_factory: &HttpFactory,
     web3: &Web3,
-) -> Result<Arc<dyn GasPriceEstimating + Send + Sync>> {
+) -> Result<Arc<dyn GasPriceEstimating>> {
     let network_id = web3.net().version().await?;
-    Ok(match gnosis_safe::api_url_from_network_id(&network_id) {
-        Some(url) => Arc::new(gnosis_safe::GnosisSafeGasStation::new(http_factory, url)?),
-        None => Arc::new(web3.clone()),
-    })
+    let mut estimators = Vec::<Box<dyn GasPriceEstimating>>::new();
+
+    if network_id == "1" {
+        let gasnow = gasnow::GasNow::new(http_factory)?;
+        estimators.push(Box::new(gasnow));
+
+        let ethgasstation = ethgasstation::EthGasStation::new(http_factory)?;
+        estimators.push(Box::new(ethgasstation));
+    }
+
+    if let Some(gnosis_url) = gnosis_safe::api_url_from_network_id(&network_id) {
+        let gnosis_estimator = gnosis_safe::GnosisSafeGasStation::new(http_factory, gnosis_url)?;
+        estimators.push(Box::new(gnosis_estimator));
```

Comment:
Why is GnosisSafe the only gas station that requires a URL? Is it dynamic? The others use the `http_factory`, but aren't given a URL?

Reply:
The gnosis one uses different urls for rinkeby and mainnet while the gasnow and ethgasstation only support mainnet (so there is no configurable url).